exp3/run_recognition.py

```
@@ -123,25 +123,27 @@ def main():
     print(f"  识别准确率: {accuracy:.2f}%")
     
     # 打印各数字的识别准确率
-    print(f"\n各数字识别准确率:")
-    for digit in config.DIGITS:
-        digit_total = len(test_files[digit])
-        digit_correct = confusion_matrix[digit][digit]
-        digit_accuracy = (digit_correct / digit_total * 100) if digit_total > 0 else 0
-        print(f"  数字 {digit}: {digit_correct}/{digit_total} ({digit_accuracy:.2f}%)")
+    print(f"\n各标签识别准确率:")
```

Ruff (0.14.5)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)